include/OpenSpaceToolkit/Astrodynamics/Trajectory/Segment.hpp

```diff
@@ -37,7 +43,7 @@ using ostk::astro::Dynamics;
 using ostk::astro::dynamics::Thruster;
 using ostk::astro::EventCondition;
 
-/// @brief                                  Represent a propagation segment for astrodynamics purposes
+/// @brief                      Represent a propagation segment for astrodynamics purposes
 
 class Segment
```

Comment:
should all of these return functions, `getDynamics...` but also `computedeltaV` etc. return `const`?

e.g. `const MatrixXd getDynamics...`

Reply:
So usually the philosophy for OSTk is to have getters and compute functions not return `const` and if wanted, implement a `access*` function which would return `const` type that then can't be modified. We can add accessors in the future on top if wanted, see
```

        const Instant& accessStartInstant() const;

        /// @brief              Access end instant
        /// @return             End Instant

        const Instant& accessEndInstant() const;

        /// @brief              Get initial mass
        /// @return             Initial mass

        Mass getInitialMass() const;

        /// @brief              Get final mass
        /// @return             Final mass

        Mass getFinalMass() const;

        /// @brief              Get propagation duration
        /// @return             Propagation duration
```
for instance